### ISSUE TITLE
How to build by Android Studio?

### ISSUE THREAD

**Issue by lhm12520** [libretro/RetroArch#7703]
Hello，I am a newcomer，how can i build by Android Studio?

**Comment by bparker06** [comment]
We only support building from command-line via ant.

**Comment by orbea** [comment]
I'm going to close this as unsupported, please see the docs for building on android.

https://docs.libretro.com/compilation/android/

**Comment by momashi** [comment]
The docs are seriously out of date. Nobody uses it ant this point, RetroArch have even move the ant build to a legacy folder.

**Comment by twinaphex** [comment]
Yes, that is correct. Unfortunately, the guy that helped us migrate now to Gradle/Android Studio doesn't feel like he has what it takes to write the documentation.

**Comment by ds17f** [comment]
Does anybody have any info on this?  Is anybody successfully building with Android Studio?  I'd love to get the debugger working and to be able to instruct others on how to do that.  I'd be happy to write documentation if somebody would support me in getting it working.
